README.md

```diff
@@ -3,7 +3,14 @@ The repository for all things DD-CMS as part of the Integrated Data Service proj
 
 A training on how to create your own website using Volto is available as part of the Plone training at [https://training.plone.org/5/volto/index.html](https://training.plone.org/5/volto/index.html).
 
+### Quick Start
+
+To run the project with default behaviour, run:
```

Comment:
This is mixed up with the other PR so probably doesn't need addressing here, but this is non default behaviour, i.e use a local file store. We'll probably need to update this docker compose when the other PR is working.